lib/cf/bridge/src/index.js

```diff
@@ -293,7 +293,9 @@ const stopReporting = () => {
 };
 
 const scheduleUsageReporting = () => {
-  oauth.start('bridge', 'secret');
+  var clientId = process.env.CLIENT_ID ? process.env.CLIENT_ID : "bridge";
```

Comment:
I would prefer that too, but it had hard coded values originally, I assumed they were there on purpose?